go/coding_standard.md

```diff
@@ -31,18 +31,29 @@ Please add items here, or modify them as appropriate. We have git versioning, do
  * Many comments are not sentences, these should begin with a lower case letter and end without a period.
  * The first letter of sentences in comments are capitalized and ends with a period.
  
-## Git
-
- * Production repos never push to `master` or `develop` branches except the push-master.
- * `develop` should always pass tests.
- * Always start work with a "WIP: \*" (work-in-progress) pull-request, on a branch ideally named "feature/\*" or "hotfix/\*". You're encouraged to fork the repo and add it as a remote in .git/config (so you can push to your fork, but still use the same import paths to the original project).
- * When complete, pull requests should be opened to the `develop` branch.
- * Do not merge PRs that aren't final -- e.g. no code dead code or code that won't be used should be committed.
- * Squash and merge to keep the history clean.
- * Merging from `develop` to `master` happens during a release.
- * All branches should be in lowercase, underscores are okay in branch names
- * All pull requests should include any relevant additions to the `CHANGELOG.md`, If PR is to the develop branch changes
- can be included under an `unreleased changes` header at the top of the document.
+## Github
+
+ * Production branches
+   * Production repos never push to `master` or `develop` branches except the push-master.
+   * `develop` should always pass tests.
+   * `develop` branch should have an on going open PR to `master`, high level comments can be made here
+   * Merging from `develop` to `master` happens during a release.
+ * Feature branches
+   * should following the naming convention `yourname/branch-description`
+   * all branches should be in lowercase, underscores/dashes are okay in branch names
+   * should have an open PR with the name "WIP: your feature description"
```

Comment:
I don't see the point of this. Cluttering the list of PRs with "WIP" PRs that stay there for weeks seems unprofessional. What is the benefit of a "WIP" PR?

Reply:
This shows contributors that there is active work done on a certain ticket.

> PRs that stay there for weeks 

That those stay for weeks is the problem. A PR should be closed in 2-3 days. If not the scope might not be defined well.

Reply:
@greg-szabo i agree with you on this one, but many of the devs find it helpful...

Reply:
It seems all PMs agree :) (Fabo's comment that a PR should be closed within 2-3 days tell me it's really not a WIP, but a regular PR.)

Reply:
I need to clarify: I like the "WIP" approach as it creates transparency about what is worked on, especially if the work takes longer then one day. The "WIP" flag then shows me, that I don't need to dive in and approve it yet.
But I think work on one item should in general be done after 3 days.

Reply:
It's not always the case that all features can be created in such a short period of time, however I do agree that we should keep as small parcelized as possible. I think it's absolutely essential to have WIP in the title to keep track of what is in on-going development - this has already been established for a while now... if you're working on a branch, open a PR, add WIP

Reply:
I still don't see the point of opening a PR to a branch that's not "ready" for review. (I see the point when the review says that there needs to be extra work done.) Instead of fighting semantics, I'm going to accept that this is how we do it. I hope it has real benefits for the dev team.